packages/oca/README.md

```diff
@@ -1,9 +1,9 @@
 <p align="center">
   <br />
   <img
-    alt="Hyperledger Aries logo"
-    src="https://raw.githubusercontent.com/hyperledger/aries-framework-javascript/aa31131825e3331dc93694bc58414d955dcb1129/images/aries-logo.png"
-    height="200px"
+    alt="Credo Logo"
```

Comment:
I'm not sure we should use the Credo logo in the bifold repo?

Reply:
Oh yes, sure. I misinterpreted that